projects/ui/src/Components/Structure/Header.tsx

```diff
@@ -0,0 +1,147 @@
+import { useContext } from "react";
+import { Link, NavLink } from "react-router-dom";
+import { ReactComponent as Logo } from "../../Assets/logo.svg";
+import { AppContext } from "../../Context/AppContext";
+import { AuthContext, useIsLoggedIn } from "../../Context/AuthContext";
+import {
+  appliedOidcAuthCodeConfig,
+  logoImageURL,
+} from "../../user_variables.tmplr";
+import { useInArea } from "../../Utility/utility";
+import { ErrorBoundary } from "../Common/ErrorBoundary";
+import { BasicAuthHeaderSection } from "./BasicAuth/BasicAuthHeaderSection";
+import CustomPagesNavSection from "./CustomPagesNavSection";
+import { HeaderStyles } from "./Header.style";
+import { OidcAuthCodeHeaderSection } from "./OidcAuthCodeHeader/OidcAuthCodeHeaderSection";
+
+const Header = () => {
+  const { pageContentIsWide, portalServerType } = useContext(AppContext);
+  const { isAdmin } = useContext(AuthContext);
+  const isLoggedIn = useIsLoggedIn();
+
+  const inAdminTeamsArea = useInArea(["/admin/teams"]);
+  const inAdminSubscriptionsArea = useInArea(["/admin/subscriptions"]);
+  const inAPIsArea = useInArea(["/apis", "/api-details/"]);
+  const inAppsArea = useInArea(["/apps", "/app-details/"]);
+  const inTeamsArea = useInArea(["/teams", "/team-details/"]);
+
+  return (
+    <HeaderStyles.StyledTopNavHeader aria-label="Site top level menus">
+      <HeaderStyles.StyledTopNavContent pageContentIsWide={pageContentIsWide}>
+        <div className="logoContainer">
+          <Link aria-label="Home" to="/">
+            {!!logoImageURL ? (
+              <HeaderStyles.StyledLogoImg src={logoImageURL} alt="logo" />
+            ) : (
+              <Logo />
+            )}
+          </Link>
+        </div>
+        <div className="siteNavigating">
+          <NavLink to={"/"} className={"navLink"} end>
+            Home
```

Comment:
nit: we might want to consider moving all the nav links into an array or some such above with all if/else logic outside of the render, and then just have a map here (similar to what we did for sidebar in mesh repo)